### ISSUE TITLE
New Feature as equals with case insensitivity 

### ISSUE THREAD

**Issue by bachowsky** [besimgurbuz/ng-mat-table-filter#1]
All String comparisons work perfectly with case sensitivity. However, sometimes a customer also needs a string comparison function with case insensitivity. For Example in a case like E-Trade app, the case sensitivity is not important for user, when user wants to filter by a column.

**Comment by besimgurbuz** [comment]
I agree. Putting another `input` into `MatTableFilterHeader` directive to differentiate case sensitivity could be a solution. What do you think?

**Comment by bachowsky** [comment]
When I first thought about it, I believed that two types of equality could be included in the list. However, after browsing several e-trade sites, I now think that it would actually be much more effective for the developer to make the decision as input parameter, just like you mentioned.

**Comment by besimgurbuz** [comment]
Ok, would be great to see your contribution as well. What do you say? Would you like to add this functionality?

**Comment by bachowsky** [comment]
Sure, I would like to contribute. You can assign me to this feature.

**Comment by besimgurbuz** [comment]
Great, I am waiting for your PR then.
